diff --git a/node_modules/@tanstack/query-core/src/queryObserver.ts b/node_modules/@tanstack/query-core/src/queryObserver.ts
index e4095a1..7ac0f6c 100644
--- a/node_modules/@tanstack/query-core/src/queryObserver.ts
+++ b/node_modules/@tanstack/query-core/src/queryObserver.ts
@@ -563,6 +563,7 @@ export class QueryObserver<
 
       if (typeof placeholderData !== 'undefined') {
         status = 'success'
+        // @ts-ignore
         data = replaceData(prevResult?.data, placeholderData, options) as TData
         isPlaceholderData = true
       }
