diff --git a/node_modules/@tanstack/query-core/src/queryObserver.ts b/node_modules/@tanstack/query-core/src/queryObserver.ts
index d23850b..78ecb2d 100644
--- a/node_modules/@tanstack/query-core/src/queryObserver.ts
+++ b/node_modules/@tanstack/query-core/src/queryObserver.ts
@@ -540,6 +540,7 @@ export class QueryObserver<
 
       if (typeof placeholderData !== 'undefined') {
         status = 'success'
+        // @ts-ignore
         data = replaceData(prevResult?.data, placeholderData, options) as TData
         isPlaceholderData = true
       }
